src/amo/components/Home/index.js

```diff
@@ -43,6 +45,33 @@ export const isFeaturedCollection = (
   });
 };
 
+export const getFeaturedCollectionsMetadata = (i18n: I18nType) => {
```

Comment:
This isn't a Flow-typed file so all annotations will be ignored. Thus, it's better to remove them so they don't bitrot.

Reply:
Ah, yes, I didn't notice that. Although I was surprised that no other types were imported. I guess I should have put two and two together.